backend/Infrastructure/Repositories/Agreement/AgreementDbRepository.cs

```diff
@@ -29,6 +29,21 @@ public class AgreementDbRepository(ApplicationContext context) : IAgreementsRepo
             .ToListAsync(cancellationToken);
     }
 
+    public async Task<List<int>> GetConsultantIdsRelatedToAgreementId(int agreementId, CancellationToken cancellationToken)
+    {
+        var consultantIds = await context.Agreements
+            .Include(a => a.Engagement)
+            .ThenInclude(e => e!.Staffings)
```

Comment:
Just for my sake, what does e!.Staffings mean? :D

Reply:
That is what I assumed, just wanted to be sure as I've only seen ? been used before. Thank you!